python/pyspark/pandas/sql_processor.py

```diff
@@ -65,6 +66,9 @@ def sql(
     ----------
     query : str
         the SQL query
+    index_col: str or list of str, optional
```

Comment:
is `list of str` the way in NumPy documentation style?

Reply:
Yeah I think so. It's documented in style guide for numpydoc.

Reply:
Oh, but the space before `:` is missing. I just added the space.